### PR TITLE
Draft: Added custom generator.

### PR DESCRIPTION
**Pull request by admoore0**
I've gotten exclusions to work correctly. I might add some more complicated logic for cube goals (i.e. 6 anti and 6 golden should exclude 8 total, but right now they don't).

Next steps are adding minimum distance calculations for the last goals (hopefully to reduce number of times needed to get a good board), and goal re-scaling.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.vscode
```
Comment (thearst3rd) [inline .gitignore]: Don't add this to the repo's .gitignore. Instead, add this into your local `.git/info/exclude` file

File: custom_generator.js [hunk 2]
```diff
@@ -0,0 +1,856 @@
+/**
+ * Custom bingo generator, taking into account dynamic difficulty.
+ */
+
+ var bingo_list_custom = {
```
Comment (thearst3rd) [inline custom_generator.js:5]: `bingo_list_custom` is in its own file, so it should be removed from here

Comment (admoore0) [reply custom_generator.js:5]: That's my plan for before I mark as ready. Right now having everything in one file makes it easier to debug locally.

**Comment by thearst3rd** [comment]
I've gotten it to work with your changes on some local html files. I'll merge this then the generator to use your custom one.

Thanks!!

**Pull request merged**